### DISCUSSION TITLE
Database-style merge

### DISCUSSION THREAD

**Discussion by karosc** [pydata/xarray#8275] [Q&A]
I have a timeseries dataset I am trying to map a monthly pattern onto. I would normally achieve this in pandas by using `pd.merge` to achieve a database-style *JOIN*. I cannot figure out how to do this in xarray properly. 

I have long timeseries Dataset with a `time` dim and a 12-value-long monthly pattern DataArray with a `month` dim. Can I do a database style of  these two structures to get a new dataset with a new variable that has my monthly pattern mapped to each time in my long timeseries based on the its month?

**Comment by dcherian** [comment]
`expanded = monthly.sel(month=ds.time.dt.month)`?
